DESCRIPTION

```diff
@@ -4,20 +4,36 @@ Title: COIN CBC MILP Solver Bindings
 Version: 0.1.0.9001
 Description: An R interface to the CBC solver for mixed-integer linear programs.
 Authors@R: c(
-  person("Dirk", "Schumacher", email = "mail@dirk-schumacher.net", role = c("aut", "cre")),
-  person("Jeroen", "Ooms", , "jeroen@berkeley.edu", role = "aut"),
-  person("Bulat", "Yapparov", , "bulat.yapparov@made.com", role = "ctb"))
+  person("Dirk", "Schumacher",
+         email = "mail@dirk-schumacher.net",
+         role = c("aut", "cre")),
+  person("Jeroen", "Ooms",
+         email = "jeroen@berkeley.edu",
+         role = "aut"),
+  person("Bulat", "Yapparov",
+         email = "bulat.yapparov@made.com",
+         role = "ctb"),
+  person(c("Jeffrey", "O"), "Hanson",
+         email="jeffrey.hanson@uqconnect.edu.au",
+         role = c("ctb"),
```

Comment:
I think the role should be `aut` here if you like.

Reply:
Yeah that would be great - thanks! I'll update the role to `aut` now.